src/main/java/seedu/address/logic/commands/DeleteAttributeCommand.java

```diff
@@ -0,0 +1,99 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.Prefix;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ROLE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SLACK;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TELEGRAM;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TIMEZONE;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import seedu.address.model.Model;
+import seedu.address.model.person.*;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.contact.Contact;
+import seedu.address.model.person.contact.ContactType;
+import seedu.address.model.tag.Tag;
+import seedu.address.ui.MainPanelName;
+
+public class DeleteAttributeCommand extends Command{
+    public static final String COMMAND_WORD = "delete";
+    public static final String MESSAGE_DELETE_SUCCESS = "Contact Attribute successfully deleted";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets a person's Contacts. "
+    + "Parameters: "
+    + PREFIX_EMAIL + "EMAIL "
+    + PREFIX_PHONE + "PHONE "
+    + PREFIX_SLACK + "SLACK"
+    + PREFIX_TELEGRAM + "TELEGRAM"
+    + "Example: " + COMMAND_WORD + " "
+    + PREFIX_EMAIL + "e0123456@u.nus.edu "
+    + PREFIX_PHONE + "87654321"
+    + PREFIX_SLACK + "johndoe "
+    + PREFIX_TELEGRAM + "@johndoe";
```

Comment:
I think you might have missed out `MESSAGE_USAGE`.

Reply:
Fixed.